run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+socat tcp-listen:9980,reuseaddr,fork tcp:localhost:8000 &
+
+SIAD_CMD=$(cat <<-END
```

Comment:
Is this `cat <<-END` part necessary? I think I know what it does but I haven't seen this syntax before.

Reply:
This is basically a multi-line string. I'm fairly bad at shell scripting and this was the best solution I could find. My previous jab at it was a single-line string which was simpler but way less readable.

Suggestions are welcome. :)

Reply:
I thought `\` at the end of each line was enough. I would just add a quick comment here explaining the syntax.

Reply:
That doesn't work in Bash, unfortunately.  Shell scripting is a little rigid.

The syntax is fairly standard for anything used in shell (Ruby and Perl use the same one), so I don't think we should be explaining it here. But adding a comment that explains *what* we are doing makes perfect sense - I'll add one right away.